x-pack/plugin/sql/qa/src/main/resources/datetime.sql-spec

```diff
@@ -31,10 +32,10 @@ SELECT MONTHNAME(CAST('2018-09-03' AS TIMESTAMP)) month FROM "test_emp" limit 1;
 dayNameFromStringDateTime
 SELECT DAYNAME(CAST('2018-09-03' AS TIMESTAMP)) day FROM "test_emp" limit 1;
 
-quarterSelect
+dateTimeQuarter
```

Comment:
the file name is already used as a prefix and adding dateTime in the test name becomes redundant:
{datetime.dateTimeQuarter}.

Reply:
Yep, but currently I just tried to make ti consistent with all the others: https://github.com/elastic/elasticsearch/pull/37693/files/95a0990cfcc48c1aee6fdb0ab1ff9cb569aafd3a#diff-6079120a3b84ce0d0410436f58ec8a5bR17

Reply:
I see - the others are inconsistent though :)